README.md

```
@@ -21,6 +22,33 @@ Or install it from npm, which ships the minified data plus TypeScript definition
 npm install pokemon-tcg-pocket-cards
 ```
 
+### 📥 npm entry points
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)